src/modules/navigator/rtl_params.c

```diff
@@ -134,7 +135,7 @@ PARAM_DEFINE_INT32(RTL_TYPE, 0);
  * @value 90 Only climb to at least RTL_DESCEND_ALT above destination.
  * @group Return Mode
  */
-PARAM_DEFINE_INT32(RTL_CONE_ANG, 0);
```

Comment:
Now I'm wondering why this wasn't already set by default.

Reply:
Yes that's a big wonder. I found the cone RTL always really useful so far, both for VTOL in hover and MC.